store/src/commonMain/kotlin/com/dropbox/external/store5/concurrent/StoreSecurity.kt

```diff
@@ -0,0 +1,14 @@
+package com.dropbox.external.store5.concurrent
+
+import kotlinx.coroutines.sync.Mutex
+
+internal data class StoreSecurity(
```

Comment:
Super NIT: `Security` makes me thing of permissions and hacking prevention. Can we do safety? :)

Reply:
https://github.com/MobileNativeFoundation/Store/pull/450/commits/724636652fd9838ed9621f8817dffabcef97d8fe